tests/components/whirlpool/test_init.py

```diff
@@ -9,7 +9,7 @@
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
 
-from . import init_integration, init_integration_with_entry
```

Comment:
The merge appears to have removed the import :(

Reply:
ah dammit, missed it. thanks!